test/functional/apps/management/_scripted_fields.js

```diff
@@ -330,6 +372,28 @@ export default function ({ getService, getPageObjects }) {
         await filterBar.removeAllFilters();
       });
 
+      //add a test to sort boolean
+      //existing bug: https://github.com/elastic/kibana/issues/75519 hence the issue is skipped.
+      it.skip('should sort scripted field value in Discover', async function () {
+        await testSubjects.click(`docTableHeaderFieldSort_${scriptedPainlessFieldName2}`);
+        // after the first click on the scripted field, it becomes secondary sort after time.
+        // click on the timestamp twice to make it be the secondary sort key.
+        await testSubjects.click('docTableHeaderFieldSort_@timestamp');
+        await testSubjects.click('docTableHeaderFieldSort_@timestamp');
+        await PageObjects.header.waitUntilLoadingHasFinished();
+        await retry.try(async function () {
+          const rowData = await PageObjects.discover.getDocTableIndex(1);
+          expect(rowData).to.be('Sep 18, 2015 @ 18:20:57.916\ntrue');
```

Comment:
Since this didn't work, I'm wondering if we should change these from 
`expect(rowData).to.be('Sep 18, 2015 @ 18:20:57.916\ntrue');`

to something that makes it obvious to the person unskipping the test that it needs to be updated like;
`expect(rowData).to.be('updateExpectedResultHere\ntrue');`

Reply:
aagreed.Updated.